src/main/java/tool/Command.java

```diff
@@ -0,0 +1,161 @@
+                    ui.printError("☹ OOPS!!! Follow the format: deadline [activity] /by [year]-[month]-[day] [hour]:[minute]");
+                } else {
+                    String description = arguments.split("/")[0].trim();
+                    String datetime = arguments.split("/")[1].replaceFirst("by ", "");
+
+                    String[] datetimeSplit = datetime.split(" ");
+                    String date = datetimeSplit[0];
+                    String time = datetimeSplit[1];
+
+                    LocalDate dateParsed = LocalDate.parse(date);
+                    LocalTime timeParsed = LocalTime.parse(time);
+                    LocalDateTime datetimeParsed = timeParsed.atDate(dateParsed);
+
+                    String newDateTime = datetimeParsed.format(DateTimeFormatter.ofPattern("dd MMM yyyy HH:mma"));
+                    Task newDeadline = new Deadlines(description, newDateTime);
+                    taskList.add(newDeadline);
+                    ui.print("Got it. I've added this task:");
+                    ui.print(newDeadline.toString());
+                    ui.print("Now you have " + String.valueOf(taskList.size()) + " tasks in the list.");
+                }
+                return false;
+            case "event":
+                if (!arguments.contains("/at ")) {
+                    ui.printError("☹ OOPS!!! The /at of an event cannot be empty.");
+                } else if (arguments.split(" ").length < 7) {
+                    ui.printError("☹ OOPS!!! Follow the format: event [activity] /at [year]-[month]-[day] [hour]:[minute] to [year]-[month]-[day] [hour]:[minute]");
+                } else {
+                    String description = arguments.split("/")[0].trim();
+                    String datetime = arguments.split("/")[1].replaceFirst("at ", "");
+
+                    String[] datetimeSplit = datetime.split(" ");
+                    String date1 = datetimeSplit[0];
+                    String time1 = datetimeSplit[1];
+                    String date2 = datetimeSplit[3];
+                    String time2 = datetimeSplit[4];
+
+                    LocalDate dateParsed1 = LocalDate.parse(date1);
+                    LocalTime timeParsed1 = LocalTime.parse(time1);
+                    LocalDateTime datetimeParsed1 = timeParsed1.atDate(dateParsed1);
+                    LocalDate dateParsed2 = LocalDate.parse(date2);
+                    LocalTime timeParsed2 = LocalTime.parse(time2);
+                    LocalDateTime datetimeParsed2 = timeParsed2.atDate(dateParsed2);
+
+                    String newDateTime1 = datetimeParsed1.format(DateTimeFormatter.ofPattern("dd MMM yyyy HH:mma"));
+                    String newDateTime2 = datetimeParsed2.format(DateTimeFormatter.ofPattern("dd MMM yyyy HH:mma"));
+
+                    String newDateTime = newDateTime1 + " to " + newDateTime2;
+
+                    Task newTask = new Events(description, newDateTime);
+                    taskList.add(newTask);
```

Comment:
I am not sure whether you have already refactored your codes. If not, you might need to refactor these codes with segmentation or better organization. Now, it is difficult to read your codes.